erts/doc/src/erlang.xml

```diff
@@ -9412,6 +9412,18 @@ Metadata = #{ pid => pid(),
         <p>Returns the previous system monitor settings just like
           <seealso marker="#system_monitor/0">
           <c>erlang:system_monitor/0</c></seealso>.</p>
+	<p>The arguments to <c>system_monitor/2</c> specifies how all
+	  system monitoring on the node should be done, not how it should be
+	  changed. This means only one process at a time
+	  (<c><anno>MonitorPid</anno></c>) can be the receiver of system monitor
+	  messages. Also, the way to clear a specific monitor option
+	  is to not include it in the list <c><anno>Options</anno></c>. All
+	  system monitoring will however be cleared if the process identified by
+	  <c><anno>MonitorPid</anno></c> terminates.</p>
+	<p>There are no special option values (like zero) to clear an option.
+	  Some of the options have a unspecified minimum value. Lower values
+	  will be adjusted to the minimum value. For example, it's currently not
```

Comment:
it's => it is

Our style guide for the documentation says that we should not use contractions.